client/src/components/avg_rating/avg_rating.jsx

```diff
@@ -1,7 +1,9 @@
 const React = require('react');
 
 function AvgRating(props) {
-  return <div>AVERAGE RATING</div>;
+  return <div>{props.data.length !== 0 && 
+                Math.round((props.data.map((data) => data.average).reduce((acc, cur) => acc + cur)) / props.data.length)}
```

Comment:
I realize this is probably work in progress, but just curious, are you planning on rendering this as stars or a number?